base/utils/R/utils.R

```diff
@@ -207,21 +207,21 @@ zero.bounded.density <- function(x, bw = "SJ", n = 1001) {
 ##' @author David LeBauer, Alexey Shiklomanov
 summarize.result <- function(result) {
   ans1 <- result %>%
-    dplyr::filter(n == 1) %>%
+    dplyr::filter(.data$n == 1) %>%
     dplyr::group_by(.data$citation_id, .data$site_id, .data$trt_id,
                     .data$control, .data$greenhouse, .data$date, .data$time,
                     .data$cultivar_id, .data$specie_id, .data$name, .data$treatment_id) %>%
     dplyr::summarize( # stat must be computed first, before n and mean
-      statname = dplyr::if_else(length(n) == 1, "none", "SE"),
-      stat = stats::sd(mean) / sqrt(length(n)),
-      n = length(n),
+      statname = dplyr::if_else(length(.data$n) == 1, "none", "SE"),
+      stat = stats::.data$sd(.data$mean) / sqrt(length(.data$n)),
```

Comment:
```suggestion
      stat = stats::sd(.data$mean) / sqrt(length(.data$n)),
```

We actually do want the `stats::sd` function here rather than a variable in the data